app/models/lti_course_datum.rb

```diff
@@ -1,3 +1,10 @@
 class LtiCourseDatum < ApplicationRecord
   belongs_to :course
+  attribute :membership_url, :string
```

Comment:
Is there a reason why you are defining all the attributes explicitly? I think it's safe to leave them out.

The defaults on `auto_sync` and `drop_missing_students` should probably be defined in the migration instead, e.g.

```
add_column :lti_course_data, :auto_sync, :boolean, default: false
add_column :lti_course_data, :drop_missing_students, :boolean, default: false
```

Reply:
Will remove the explicit attribute definitions and define the defaults in the migration.